ruby/ql/lib/codeql/ruby/Concepts.qll

```diff
@@ -321,6 +339,14 @@ module Http {
          * This is typically the name of the method that gives rise to this input.
          */
         abstract string getSourceType();
+
+        /**
+         * Gets the kind of the accessed input,
+         * Can be one of "parameter", "header", "body", "url", "cookie".
+         *
+         * Note that this predicate is functional.
```

Comment:
same here.

Reply:
This is gone now due to pulling in the changes from the other PR.